tests/README.md

```diff
@@ -16,7 +16,15 @@ and the code is in the `com.nvidia.spark.rapids.tests.mortgage` package.
 Unit tests exist in the [tests]() directory. This is unconventional and is done so we can run the 
 tests on the final shaded version of the plugin. It also helps with how we collect code coverage. 
 
-Use Maven to run the unit tests via `mvn test`.
+The `tests` module depends on the `aggregator` module which is shading the dependencies.
+Should first `mvn install` the aggregator jar to local Maven repository due to a limitation of the shading system.
+The steps to run the unit tests:
```

Comment:
we may want to clarify that there are other ways to run the tests... ie just packages without skipTests.
Perhaps change the wording such if trying to just run the tests vi mvn test make sure to do an install first.